parsers/IN.py

```diff
@@ -53,7 +53,6 @@
     "उत्तर-पूर्वी क्षेत्र  / North-Eastern Region": "IN-NE",
 }
 
-DEMAND_URL = "https://vidyutpravah.in/state-data/{state}"
 STATES_MAPPING = {
```

Comment:
Looks like there is a duplicate `STATES_MAPPING` as well, would you mind removing that as well before we merge this?